src/main/java/org/jboss/tools/intellij/openshift/utils/oc/OcCli.java

```diff
@@ -60,10 +63,30 @@ private static void execute(@NotNull File workingDirectory, String command, Map<
 
   @Override
   public void login(String url, String userName, char[] password, char[] token) throws IOException {
+    var args = new ArrayList<String>();
+    args.add("login");
+    args.add(url);
+    args.add("--insecure-skip-tls-verify");
+    addKubeconfigParameter(args);
```

Comment:
this is the main change here. Use `--kubeconfig` so that the contexts that are defined in the other configs files are not dumped into the first one. Only a new context for the cluster (that we're logging in) is added.